vitest.config.ts

```diff
@@ -0,0 +1,13 @@
+import { mergeConfig } from 'vite';
+import { defineConfig } from 'vitest/config';
+import viteConfig from './vite.config';
+
+export default mergeConfig(
+  viteConfig,
+  defineConfig({
+    test: {
+      setupFiles: ['src/installSesLockdown.ts'],
+      environment: 'happy-dom',
```

Comment:
Using jsdom fails on lockdown with `Unexpected intrinsic intrinsics.ArrayBuffer.__proto__`, but luckily the aptly named `happy-dom` works